specs/Renting/P2PCarRental/README.md

```diff
@@ -0,0 +1,4 @@
+# P2P Car Rental
+
```

Comment:
is this about renting or just another transportation option?

Reply:
this is about renting